tests/Integration/Client/TestCase.php

```diff
@@ -49,7 +49,12 @@ protected function assertSuccessWithFile(Response $response, DataTransferObject
     {
         if (!$response->getSuccess()) {
             $this->debug(print_r($response->getPayload()->toArray(), true));
+
+            if ($response->getPayload()->status === 500) {
```

Comment:
@snapshotpl In what cases did you encounter 500 errors?

Reply:
https://github.com/michalbiarda/shipx-php-sdk/blob/cbdd3abc3a7f957afbcd2b6bd3a8e01af714b8ba/tests/Integration/Client/ReportResourceTest.php#L18

Reply:
Do you encounter this 500 error every time you perform a request? I have never had issues with this one, nor do I have it now...

I would rather not skip test because of 500 error. Such mechanism might hide some real issues in the future.